scripts/run_examples.py

```diff
@@ -54,6 +52,19 @@ def run_examples(keep_artifacts=None):
     int
         Number of failed examples
     """
+    conda_exes = [None]
+    if platform.system() != 'Windows':
+        env_name = "constructor-testing-micromamba"
+        which_cmd = ["conda", "run", "--name", env_name, "which", "micromamba"]
+        errored, micromamba_path = _execute(which_cmd)
+        if errored:
+            _execute(["conda", "create", "--name", env_name, "--yes", "-c", "conda-forge", "micromamba"])
```

Comment:
Wouldn't this be checking latest micromamba version? Usually new micromamba releases break something and this would make constructor break too. (Unless micromamba tests itself against constructor)

Reply:
> Wouldn't this be checking latest micromamba version?

Yes it is, though I think it's good to catch those breaks here. As it's maturing I hope micromamba's test suite is becoming good enough to minimise the amount of regressions for constructor.

Reply:
There's no test at all in micromamba for constructor which is why there are constant failures in miniforge.
I think it's good to have a test here, but it should be after a constructor test has been added to micromamba.

Reply:
I mean testing the behavour `constructor` expects rather than running `constructor` explicitly. For example:

* https://github.com/mamba-org/mamba/blob/d203a671de38b79f13fb90a6cdbd4443ec931892/libmamba/tests/test_url.cpp#L249-L254
* https://github.com/mamba-org/mamba/blob/d203a671de38b79f13fb90a6cdbd4443ec931892/libmamba/tests/test_cpp.cpp#L511-L550

Reply:
can we pin the micromamba version and auto update it via auto update PRs?